xwayland/xwm.c

```diff
@@ -89,6 +91,15 @@ const char *const atom_map[ATOM_LAST] = {
 	[NET_CLIENT_LIST_STACKING] = "_NET_CLIENT_LIST_STACKING",
 };
 
+#define STARTUP_INFO_BUF_LEN 20
```

Comment:
Where is this coming from? I don't see it in the spec.

Reply:
https://gitlab.freedesktop.org/xorg/proto/xcbproto/-/blob/master/src/xproto.xml#L1032

Reply:
Maybe consider using `sizeof(ev->data)` instead?